### ISSUE TITLE
Clients can't connect anymore to Swoole websocket server with 4.4.17 (4.4.16 works)

### ISSUE THREAD

**Issue by gcmartin** [swoole/swoole-src#3232]
1. What did you do? If possible, provide a simple script for reproducing the error.

I'm creating a websocket server with this configuration:

```
$server = new \swoole_websocket_server(
    "0.0.0.0",
    8443,
    SWOOLE_PROCESS, SWOOLE_SOCK_TCP | SWOOLE_SSL
);
$server->set([
    'backlog' => 1000, // TCP max backlog connections
    'chroot' => realpath(__DIR__ . '/..'),
    'daemonize' => false,
    'enable_reuse_port' => true,
    'max_conn' => 10000,
    'max_request' => 100000,
    'open_tcp_nodelay' => true,
    'ssl_cert_file' => $sslDir . '/fullchain.pem',
    'ssl_key_file' => $sslDir . '/privkey.pem',
    'task_max_request' => 100000,
    'task_worker_num' => 4,
    'tcp_fastopen' => true,
    'timeout' => 10,
    'worker_num' => 8,
]);
```

2. What did you expect to see?

Everything works fine with Swoole 4.4.16 but users can't connect anymore with 4.4.17. 
I've made no change to the code, only updated Swoole from 4.4.16 to 4.4.17.
Everything works fine again when downgrading back to 4.4.16.

3. What did you see instead?

Connections fail with the following errors:

`>--WARNING>swSSL_accept: bad SSL client[***:***:***:***:*****], reason=193, error_string=error:000000C1:lib(0):func(0):reason(193)`

4. What version of Swoole are you using (show your `php --ri swoole`)?

4.4.17

5. What is your machine environment used (including version of kernel & php & gcc) ?

CentOS Linux release 7.7.1908
Kernel 3.10.0-1062.18.1.el7.x86_64
PHP 7.3.16
No GCC, using php-pecl-swoole4-4.4.17-1.el7.remi.7.3 package

Thanks!

**Comment by twose** [comment]
can you try to install the latest version of v4.4.x:

https://github.com/swoole/swoole-src/tree/v4.4.x

I submitted a patch just now

**Comment by gcmartin** [comment]
Thanks for the patch.

I'm using the php-pecl-swoole4 Remi RPM package, so I've patched the 4.4.17 SRC RPM with your last commit, recompiled it and everything works fine again. :)

**Comment by breath-co2** [comment]
同样问题，ssl的tcp服务，其它语言可以，golang连不上
